website/content/docs/nia/enterprise/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+page_title: Consul-Terraform-Sync Enterprise
+description: >-
+  Consul-Terraform-Sync Enterprise
```

Comment:
To clarify CTS is offered as an OSS binary and an Enterprise binary, similar to Consul OSS vs. Enterprise. This Enterprise section of the documentation is intended to encapsulate all enterprise related-information, such as licensing and enterprise features. This overview page is intended to give an overview of CTS enterprise - similar to [Consul Enterprise's overview page](https://www.consul.io/docs/enterprise).

Please let me know if that helps clarify the confusion

Reply:
Gotcha! I agree that the Overview page itself doesn't provide much additional useful information. We decided to have this enterprise Overview page to since it aligns with how some of our other products' docs are set up e.g. Consul (more importantly) and Vault. I personally like having a landing page for the general enterprise section and having "License" in the side bar as a clear signpost for users looking for license information.

Do you have any additional thoughts? If we merge the two pages, would you recommend it be named "Overview" or "License"?

Reply:
I see what you mean about having a landing page for CTS+ent because it would follow the docs template we employ. I still think that in this case the user experience outweighs the don't think there's enough content to merit it. In fact, I just realized that the TF Cloud section is the same as the page under Network Drivers. 

My recommendation is to completely flatten the content into "Consul-Terraform-Sync Enterprise" -- no child pages. On that page are the following topics:

* Overview
* Requirements (statement about what's different from OSS and link to /nia/installation/requirements) 
* Installation (statement about what's different from OSS and link to /nia/installation/install) 
* License
* * Setting the License
* * * Notification and Termination Behavior
* Configuration (statement about what's different from OSS and link to /nia/configure)
* * Network Drivers (link to the respective drivers pages)
* Usage (statement about what's different from OSS and link to /nia)

I think that would cover it.

Reply:
Thanks for identifying additional parts of the enterprise experience that aren't called out e.g. the differences in configuring and requirements. We are actually in the process of revamping our docs overall. We'll likely revamp the enterprise content at part of the overall revamp since we've heard some other feedback as well. I'll make sure to link these recommendations so that they can be considered as part of the redesign.

Since we're aiming to release soon, I'd like to merge this PR with the current enterprise docs organization. After the release, we can iterate and improve the enterprise documentation as part of the overall documentation revamp rather than designing this twice. Please don't hesitate to let me know though if this sounds concerning to you.

To the specifics of your comment:
 - Yes, I recognize that it's not ideal and potentially confusing that the same TFC docs are in two places. This was borrowed from some existing patterns.
 - One reason we decided to have the enterprise directory was to setup for future enterprise features where each feature could have a corresponding page within the Enterprise directory.

API Tooling team, if you have differing opinions, please feel free to share.